src/Components/Onboarding/Steps/Artists/ItemLink.tsx

```diff
@@ -82,7 +82,7 @@ class ItemLink extends React.Component<Props, State> {
               node {
                 __id
                 related {
-                  suggested(first: 1) @connection(key: "ItemLink_suggested", filters: []) {
```

Comment:
I didn’t really see a need to tell Relay to treat this as a ‘connection’, which I assume is mostly for things like pagination etc.

Reply:
This was bc in the struggle to try to update the Relay store, I thought it would be easier and make more sense (since docs/examples were not helping) to do it the 'Relay' way, so i was trying to make everything a connection, that's also why I did https://github.com/artsy/metaphysics/pull/802 (adding an artist edge to the follow mutation payload).

Reply:
Gotcha. I don’t think it should really matter, you can probably try using the non-edge version and I assume it will work the same.